tests/Common/FilesTest.php

```diff
@@ -541,13 +541,6 @@ public function testFilesPermissions()
         $files = $this->_client->listFiles();
         $this->assertEquals($newFileId, reset($files)['id']);
 
-        $this->_client->dropToken($newTokenId);
```

Comment:
Tuhle část jsem moc nepochopil, proč tam byla :confused:

Reply:
to ja taky ne